docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes-provider.asciidoc

```diff
@@ -83,6 +88,8 @@ Example:
         node:
           include_labels: ["nodelabel2"]
           include_annotations: ["nodeannotation1"]
+        deployment: true
+        cronjob: true
```

Comment:
```suggestion
        #cronjob: false
```

Reply:
Usually we put the default values available.

Reply:
> Usually we put the default values available

does it make sense then to comment it out if default values are set in example?

Reply:
Like here: https://github.com/elastic/beats/blob/main/metricbeat/metricbeat.yml#L27
Where the default value is false and it commented out to show that possible can be used from user

Reply:
ok, I don't really have strong preference